### DISCUSSION TITLE
Elapsed time on Ender 6

### DISCUSSION THREAD

**Discussion by alegnn** [Desuuuu/klipper#148] [Q&A]
Hi all,

First of all thanks for this great project ! 

I successfully run an Ender 6 with its screen and it works great. The only thing that I don't get is the elapsed time on the screen that seems to be shown in day units ? Is there a setting to show it in a more readable way ?  

Have a great day ! 

![IMG_20230504_105238](https://user-images.githubusercontent.com/132557306/236156930-ee3c4323-92cb-4092-b246-ecc320d80758.jpg)

**Comment by Jonnyjoebo** [comment]
Also having the same issue now. I had it working before with correct elapsed time. But re installed everything trying to get an accelerometer hooked up. Did you ever figure it out?

Also I noticed now in fluidd I only get slicer time when before I was getting 3 different times. Something like slicer time | estimated actual time | elapsed time.

I’ll try reinstalling like I did before. I think some unmodified klipper files were mixed into the modified klipper.

**Comment by Desuuuu** [comment]
The [latest release](https://github.com/Desuuuu/klipper/releases/tag/dgus-reloaded-230526) should address this issue.

**Reply by Jonnyjoebo** [reply]
Solved. Thank you!
